src/api/image/Dockerfile

```
@@ -1,6 +1,6 @@
 ## Base ###########################################################################
 # Set up the base layer
-FROM node:16-alpine3.15 as base
+FROM node:16 as base
```

Could we fix the issue by specifying the architecture on which the images will run?
```suggestion
FROM --platform=linux/amd64 node:16-alpine3.15 as base
```

---

If I do this, I'm forced to use `amd64` vs. `arm64` on my Mac.

---

How is the build time on your M1?

---

It's good.  But if I have to emulate twice it's not ideal (Docker is already in Linux and then if I have to add amd64/x86 vs. arm64, that slows things down at runtime).